lib/mix/lib/mix/tasks/app.tree.ex

```diff
@@ -0,0 +1,70 @@
+defmodule Mix.Tasks.App.Tree do
+  use Mix.Task
+
+  @shortdoc "Analyses the current Mix project and prints dependency tree on terminal."
```

Comment:
Have thought, but do not know, how really reword it. What would be right wording for it?